packages/formstr-app/src/containers/CreateForm/FormBuilder.tsx

```diff
@@ -14,8 +15,17 @@ function FormBuilder() {
   const {
     isRightSettingsOpen,
     closeOnOutsideClick: closeSettingsOnOutsideClick,
+    setFormTempId,
+    formTempId,
   } = useFormBuilderContext();
 
+  useEffect(() => {
+    if (formTempId === "") {
```

Comment:
we should move this to provider container and assign form id as the initial value while creating state

Reply:
i tried it, problem is, since it's not a static id, it regenerates the id every time, which beats the point

Reply:
sorry, must be a mistake on my end, tried it again and it works, changed it to this behaviour